solutions/standard/main.tf

```diff
@@ -431,20 +431,19 @@ module "secrets_manager_service_credentials" {
 # Code Engine Kibana Dashboard instance
 ########################################################################################################################
 
-locals {
-  code_engine_project_id   = var.existing_code_engine_project_id != null ? var.existing_code_engine_project_id : null
-  code_engine_project_name = local.code_engine_project_id != null ? null : var.prefix != null ? "${var.prefix}-code-engine-kibana-project" : "ce-kibana-project"
-  code_engine_app_name     = var.prefix != null ? "${var.prefix}-kibana-app" : "ce-kibana-app"
-  es_data                  = var.enable_kibana_dashboard ? jsondecode(data.http.es_metadata[0].response_body) : null
-  es_full_version          = var.enable_kibana_dashboard ? (var.elasticsearch_full_version != null ? var.elasticsearch_full_version : local.es_data.version.number) : null
-}
-
 data "http" "es_metadata" {
   count       = var.enable_kibana_dashboard ? 1 : 0
   url         = "https://${local.elasticsearch_username}:${local.admin_pass}@${local.elasticsearch_hostname}:${local.elasticsearch_port}"
   ca_cert_pem = base64decode(local.elasticsearch_cert)
 }
 
+locals {
+  code_engine_project_id   = var.existing_code_engine_project_id != null ? var.existing_code_engine_project_id : null
+  code_engine_project_name = local.code_engine_project_id != null ? null : var.prefix != null ? "${var.prefix}-code-engine-kibana-project" : "ce-kibana-project"
```

Comment:
There seems to be an issue with the logic here  - the CE project name is being returned as `null`  even when a valid CE project ID is provided. Please check.